scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/TerminationData.java

```diff
@@ -153,7 +163,8 @@ public void run() {
         }
     }
 
-    public Map<String, Serializable> getStringSerializableMap(SchedulingService service, TaskTerminationData taskToTerminate) throws Exception {
+    public Map<String, Serializable> getStringSerializableMap(SchedulingService service,
+            TaskTerminationData taskToTerminate) throws Exception {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00112)